web-app/src/utilities/holidayStatus.js

```diff
@@ -2,4 +2,5 @@ export default {
   PENDING: 1,
   APPROVED: 2,
   REJECTED: 3,
+  WFH: 4,
```

Comment:
I'd maybe change this ID to equal 10, in case we need to add more holiday statuses, and just to create a bit of seperation between a holiday and wfh event.

Reply:
This is all going to change very soon anyway - I imagine it will be working off an `eventType` constant instead to accommodate working remotely vs a holiday. 

I think it's fine for now - 10 seems as arbitrary as 4 - this PR is just to show how remote working could be worked into/displayed on the calendar.

Reply:
Cool, makes sense